tests/conftest.py

```
@@ -1,7 +1,46 @@
+
+    The repository's pytest configuration includes ``--cov`` and ``--cov-report``
+    flags by default. In environments where the ``pytest-cov`` plugin is not
+    available those arguments would normally cause pytest to exit with an
+    ``unrecognized arguments`` error. To keep the default configuration working
+    everywhere we register lightweight stand-ins for those options when the real
+    plugin cannot be imported. The options are accepted but ignored, which
+    mirrors the behaviour of simply running the tests without coverage enabled.
+    """
+
+    try:
+        import pytest_cov  # type: ignore  # noqa: F401
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)